src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenStringConcat.cs

```diff
@@ -767,7 +767,7 @@ .maxstack  3
   IL_0007:  brtrue.s   IL_000b
   IL_0009:  pop
   IL_000a:  ldnull
-  IL_000b:  call       ""string string.Concat(string, string)""
+  IL_000b:  call       ""string string.Concat(string?, string?)""
```

Comment:
We should drop annotations from the symbols in IL, to avoid unnecessary changes.